lib/src/components/NxList/types.ts

```diff
@@ -0,0 +1,113 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import { HTMLAttributes, LiHTMLAttributes, WeakValidationMap, ReactElement,
+  ReactNode, ButtonHTMLAttributes, AnchorHTMLAttributes } from 'react';
+import PropTypes from 'prop-types';
+
+export type NxListProps = HTMLAttributes<HTMLUListElement> & {
+  bulleted?: boolean;
+  children?: ReactElement | ReactElement[] | null | ReactNode;
+};
+
+export const nxListPropTypes: WeakValidationMap<NxListProps> = {
```

Comment:
Use `ValidationMap` instead of `WeakValidationMap`.  `WeakValidationMap` doesn't work right.

Reply:
> Use `ValidationMap` instead of `WeakValidationMap`. `WeakValidationMap` doesn't work right.

Changed to use `ValidationMap`
https://github.com/sonatype/sonatype-react-shared-components/pull/451/files#diff-acef640fb1686ef46c36378eb99744a098e85ffac1cacbcea722e14f8e0f0ce3R4-R101